<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IonogramViewer2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>497</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonE">
       <property name="toolTip">
        <string notr="true">The E trace data group</string>
       </property>
       <property name="text">
        <string notr="true">E</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditE">
       <property name="toolTip">
        <string notr="true">The critical frequency of the layer E
(Right-click)</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetE">
       <property name="toolTip">
        <string notr="true">The trace points (f in MHz, h' in km)
(Left-click)</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonF1">
       <property name="toolTip">
        <string notr="true">The F1 trace data group</string>
       </property>
       <property name="text">
        <string notr="true">F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditF1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string notr="true">The critical frequency of the layer F1
(Right-click)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetF1">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string notr="true">The trace points (f in MHz, h' in km)
(Left-click)</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonF2">
       <property name="toolTip">
        <string notr="true">The F2 trace data group</string>
       </property>
       <property name="text">
        <string notr="true">F2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditF2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string notr="true">The critical frequency of the layer F2
(Right-click)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidgetF2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string notr="true">The trace points (f in MHz, h' in km)
(Left-click)</string>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionFirst"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionReload"/>
   <addaction name="actionNext"/>
   <addaction name="actionLast"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>images/application-exit.png</normaloff>images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit (Ctrl+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>images/document-open.png</normaloff>images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>images/help-contents.png</normaloff>images/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>images/help-about.png</normaloff>images/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Alt+F1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>images/document-save-as.png</normaloff>images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset>
     <normaloff>images/go-next.png</normaloff>images/go-next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset>
     <normaloff>images/go-next-rtl.png</normaloff>images/go-next-rtl.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="toolTip">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Space</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset>
     <normaloff>images/view-refresh.png</normaloff>images/view-refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="icon">
    <iconset>
     <normaloff>images/go-first.png</normaloff>images/go-first.png</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset>
     <normaloff>images/go-last.png</normaloff>images/go-last.png</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
